Externals/expr/include/expr.h

```diff
@@ -468,6 +479,17 @@ extern "C" {
       }
       return i;
     }
+    else if (c == '"') {
+      i++;
```

Comment:
Following the behavior of the lexer for number literals, the first lines here should be something like the following:
```
    if ((*flags & EXPR_TSTRING) == 0) {
      return -6; // unexpected string
    }
    *flags = EXPR_TOP | EXPR_TCLOSE;
```